core/jazz_ui/src/index.html

```diff
@@ -95,6 +93,14 @@
             <div class="inner three"></div>
         </div>
     </app-root>
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+        
+        ga('create', 'UA-121896506-3', 'auto');
```

Comment:
Adding documentation (with code snippets) on how to integrate google analytics plugin to this code is fine but adding this by default might not be a good idea.